compiler/rustc_passes/src/check_attr.rs

```diff
@@ -1346,10 +1386,8 @@ impl<'tcx> CheckAttrVisitor<'tcx> {
                             self.check_attr_crate_level(attr, meta, hir_id);
                         }
 
-                        sym::cfg_hide => {
```

Comment:
Should we delete `sym::cfg_hide` as well? Or is it used somewhere else?

Reply:
As far as I can see, it's not used anywhere else.